pkg/operator/ceph/csi/spec.go

```diff
@@ -370,8 +370,15 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 			return errors.Wrap(err, "failed to load nfs provisioner deployment template")
 		}
 	}
-	multusApplied := len(r.multusClusters) > 0
 
+	multusApplied := false
```

Comment:
Looks like all the usages of `multusApplied` should just be replaced by the new holderEnabled check, right? So it seems we don't need the `multusApplied` var anymore, and consolidate line 390 here.
```suggestion
	holderEnabled := !CSIParam.EnableCSIHostNetwork
```

Reply:
We still need this one as we don't care about the value of `CSIParam.EnableCSIHostNetwork` if its a multus enabled ceph cluster. As the default value of `CSIParam.EnableCSIHostNetwork` is true and ceph cluster is multus enabled. this helps to identify whether to create a holder pod or not.

Reply:
Yes correct, removed extra variable